builtin/providers/google/resource_google_project.go

```diff
@@ -242,6 +172,7 @@ func resourceGoogleProjectRead(d *schema.ResourceData, meta interface{}) error {
 func prefixedProject(pid string) string {
 	return "projects/" + pid
 }
+
 func resourceGoogleProjectUpdate(d *schema.ResourceData, meta interface{}) error {
```

Comment:
I *think* it's OK, because it's not creating a new resource, then trying to assign that resource, which leaves the possibility of the second call failing and leaving the resource dangling. But I'd have to check my notes on that (SetPartial leaves me fuzzy as to when exactly it's _necessary_, so I don't want to be loudly wrong here.) Want to chat about it on Thursday?